openstack-details

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+utility_container=`lxc-ls | grep utility`
+
+echo "============================================================="
+echo "openrc"
+echo "============================================================="
+set -x
+lxc-attach -n $utility_container cat /root/openrc | tee ~/.ostrich/openrc
```

Comment:
So in newton and greater, there's ./scripts/os-cmd that makes this easier. I'm not sure if the mitaka case is enough to keep this as is, or whether you should replace all the commands below with:  "./scripts/os-cmd openstack catalog list" etc